### PR TITLE
Update examples, documents and 100% test coverage

### PR DESCRIPTION
**Pull request by tguo9**

### CODE REVIEW

File: tests/testthat/test-generate_virtual_pop.R [hunk 1]
```diff
@@ -23,3 +23,9 @@ test_that('Check the distribution', {
   helper_correct_values <- rexp(1000, 5)
   expect_equal(helper_object1, helper_correct_values)
 })
+
+test_that("Input errors no catching", {
```
Comment (hwilliams10) [inline tests/testthat/test-generate_virtual_pop.R]: Minor wording suggestion
```suggestion
test_that("Check that input errors are being caught as expected", {
```

File: tests/testthat/test-stat_summary.R [hunk 2]
```diff
@@ -1,5 +1,23 @@
...
   expect_equal(names(stat_summary(dplyr::tibble(x = 1:10), dplyr::tibble(x = 1:10), c('mean', 'sd'))), c('data', 'mean', 'sd'))
 })
+
+test_that("The population you passed in is not a valid data frame", {
+  expect_error(stat_summary(c(1, 2), dplyr::tibble(x = 1:3), 'mean'))
+  expect_error(stat_summary(dplyr::tibble(a = character()), dplyr::tibble(x = 1:10), c('mean', 'sd')))
+  expect_error(stat_summary(dplyr::tibble(a = numeric()), dplyr::tibble(x = 1:10), c('mean', 'sd')))
+})
+
+test_that("The psamples you passed in is not a valid data frame", {
```
Comment (hwilliams10) [inline tests/testthat/test-stat_summary.R]:
```suggestion
test_that("The samples you passed in is not a valid data frame", {
```

File: tests/testthat/test-stat_summary.R [hunk 3]
```diff
@@ -1,5 +1,23 @@
-test_that("multiplication works", {
+test_that("The tibble not generate as expected", {
```
Comment (hwilliams10) [inline tests/testthat/test-stat_summary.R]:
```suggestion
test_that("The tibble does not generate as expected", {
```

**Review by hwilliams10 — APPROVED** [review]
Great job! Just a couple typo / minor wording suggestions!

File: tests/testthat/test-stat_summary.R (hunk 2 above)
Comment (tguo9) [reply tests/testthat/test-stat_summary.R]: My silly typo

File: tests/testthat/test-stat_summary.R (hunk 3 above)
Comment (tguo9) [reply tests/testthat/test-stat_summary.R]: My terrible mistakes

**Pull request merged**